tests/Functional/Driver/IBMDB2/StatementTest.php

```diff
@@ -36,4 +37,16 @@ public function testExecutionErrorsAreNotSuppressed(): void
         $this->expectException(StatementError::class);
         $stmt->execute([[]]);
     }
+
+    public function testBlobBindingDoesNotOverwritePrevious(): void
```

Comment:
It would be nice to move this under `Doctrine\DBAL\Tests\Functional\BlobTest` to make sure that this scenario is handled properly on all platforms.